app/tetwild/common.h

```diff
@@ -12,12 +12,13 @@
 #include <vector>
 
 #include <Eigen/Core>
+#include <Rational.hpp>
 
 namespace tetwild {
 #define MAX_ENERGY 1e50
```

Comment:
Is this used specifically? Can we avoid this?

Reply:
what do you mean?

Reply:
1e50 is an arbitrary number, and far from std::numeric_limits<double>::max()